### ISSUE TITLE
AGP 8.0 and using ressource ids in annotations

### ISSUE THREAD

**Issue by Tolriq** [joaomgcd/TaskerPluginSample#19]
So as discussed by mail a few times, Google now starting with AGP 8.0 alpha 10 enforce this.

And you can no more compile tasker plugins.

For the moment it's still possible to opt out with
`android.nonFinalResIds=false` in gradle.properties but for how long.

It's also necessary to disable lint rule: `NonConstantResourceId`

This was delayed for a long time, but it's really time to think about alternative solution for the future.

**Comment by joaomgcd** [comment]
Sorry, I should've mentioned. You can already use resource Ids as strings with the latest update :) Can you please try that? Thanks in advance!

**Comment by Tolriq** [comment]
You always forget to push to Github :p 

Will try that in the coming weeks thanks. Does it require a specific Tasker version or 100% internal to the lib?

**Comment by joaomgcd** [comment]
Hhmm, but it's on Github too 😅

https://github.com/joaomgcd/TaskerPluginSample/commit/1b326960a8c271cde9e2da36053968735d68489b#diff-d67fd9793dc8662b4ced6d48c86d8aefb24b2bb8d2b55f12a79ac4a1b2407b4f

It's internal to the lib so Tasker version doesn't matter.

Hope this helps!

**Comment by Tolriq** [comment]
Yes it's present now :) But when you released the library to maven it was not :p Anyway no big deal.
